### PR TITLE
[#507] Screenshot updated, maintainer added

### PR DESCRIPTION
**Pull request by AakashMallik**
#507

* **Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
- [ ] Bug fix
- [ ] Feature implementation
- [x] Doc updates

* **What changes have you introduced?**
- Updated docs

* **Does this PR introduce a breaking change?**
No

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,35 +31,50 @@ A number of instruments have already been implemented.
...
 
 ### 7. Robotic arm
 A neat UI that can be used to program four servo motors ( placed in a robotic arm ) using a timeline.
-<img src="./docs/images/robot.jpg" width="700px"/>
 
 ### 8. Data logger
 Data can be captured and saved in a CSV file. We also have a playback feature in place that lets the user replay the data from the CSV file.  
-<img src="./docs/images/data_logger.jpg" width="700px"/>
+
+## Screenshots
```
Comment (mariobehling) [inline README.md]: Please add the description from above directly into the table below the screenshot image. No need to make it into headings though.

Comment (AakashMallik) [reply README.md]: Done

Comment (mariobehling) [reply README.md]: Where? There is no description - right below the images? Please also ensure the table fits into the width of the GitHub display box.

**Comment by AakashMallik** [comment]
Oh.. I thought u meant description in the table below the images... I got it... I l'll add captions to the images

**Comment by AakashMallik** [comment]
@mariobehling Screenshot description added 
screenshot-localhost_6419-2019 08 29-23_32_14

If this looks fine, please merge it.

**Pull request merged**